### DISCUSSION TITLE
Haiku is a X-line poem with Y syllables, written in N-P-N syllable count?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1007] [Galaxy Trivia] [answered]
**Bonus:** Write a haiku about your ❤️ programming language.

**Comment by peterbaccaro** [comment]
Haiku is a **3**-line poem with **17** syllables, written in **5**-**7**-**5** syllable count.

**Indentation rules,**
**Pythonic elegance flows,**
**Code snakes dance gracefully.**

**Comment by Luanf** [comment, marked as answer]
X = 3 (Haiku is a traditional form of Japanese poetry consisting of three lines)
Y = 17 (The entire poem contains 17 syllables)
N = 5 and P = 7 (Haiku poems typically have a syllable count of 5 for the first line, 7 for the second line, and 5 for the third line)

**JavaScript dances,
In each line, a world unfolds,
Code that shapes our dreams.**

**Comment by thuongtruong1009** [comment]
Code flows like a stream,
Syntax dances with logic,
Creating my world.
